### PR TITLE
Added backups support. We can add scheduled backups and(or) PITR(Point in time recovery)

### PR DESCRIPTION
**Pull request by CyberCuCuber**

### CODE REVIEW

File: modules/gcp/firestore/db/input.tf [hunk 1]
```diff
@@ -44,3 +44,35 @@ variable "deletion_policy" {
...
   default     = "ABANDON"
 }
+
+variable "pitr" {
+  description = "Point in time recovery. If True - enabled"
+  type        = bool
+  default     = false
+}
+
+variable "backup" {
```
Comment (ggramal) [inline modules/gcp/firestore/db/input.tf:54]: we try to use objects as much as possible in this modules. Can we switch to something like
```
backup = {
  enabled = true
  retention = "10080s"
  weekly = true
  /* OR */
  daily = true
  weekly_recurrence_day = "SOME DAY"
}
```

Comment (CyberCuCuber) [reply modules/gcp/firestore/db/input.tf:48]: ok. Сheck my last commits. I have corrected and added the ability to use both daily and weekly backups.

File: modules/gcp/firestore/db/main.tf [hunk 2]
```diff
@@ -18,4 +19,24 @@ resource "google_app_engine_application" "this" {
 }
 
 
+resource "google_firestore_backup_schedule" "daily-backup" {
```
Comment (ggramal) [inline modules/gcp/firestore/db/main.tf:22]: can we use single resource of `google_firestore_backup_schedule`

Comment (CyberCuCuber) [reply modules/gcp/firestore/db/main.tf:22]: If we use both backups (daily and weekly), we need use 2 resources.

Comment (ggramal) [reply modules/gcp/firestore/db/main.tf:22]: ok got it

**Review by ggramal — APPROVED** [review]

**Pull request merged**